php/api/controller/class-audit-posts-controller.php

```diff
@@ -342,6 +343,31 @@ public function prepare_item_for_response( $post, $request ) {
 			$response->remove_link( $field );
 		}
 
+		// Only for authenticated users.
```

Comment:
Inside [`Rate_Limit::identify_client()`](https://github.com/wptide/wp-tide-api/blob/develop/php/restriction/class-rate-limit.php#L207-L241) we handle this logged in check with a fallback. Perhaps we should add something like the following to ensure the JWT user is also checked:
```
$is_user_logged_in = is_user_logged_in();
if ( false === $is_user_logged_in ) {
	$is_user_logged_in = get_user_by( 'id', Plugin::instance()->components['jwt_auth']->get_user_id() );
}
```